### PR TITLE
Removed Kernel32 library use from HidDevice. "As Tested" on-site and "approved" vers…

### PR DESCRIPTION
**Pull request by kuznetsovmoci**
…ion.

### CODE REVIEW

File: src/purejavahidapi/windows/HidDevice.java [hunk 1]
```diff
@@ -165,30 +164,37 @@ synchronized public int setOutputReport(byte reportID, byte[] data, int length)
...
 					// register_error(dev, "WriteFile");
+					System.err.println("WriteFile failed with GetLastError()==" + GetLastError());
 					return -1;
 				}
 			}
 
-			if (WAIT_OBJECT_0 != Kernel32.INSTANCE.WaitForSingleObject(m_OutputReportOverlapped.hEvent, 1000)) {
-				return -2;
+			if (WAIT_OBJECT_0 != WaitForSingleObject(m_OutputReportOverlapped.hEvent, 1000)) {
+				// This could be an overkill and not needed. When there is a chance need to be tested without this block
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/HidDevice.java:191]: Does this need to be resolved before merging?

File: src/purejavahidapi/windows/HidDevice.java [hunk 2]
```diff
@@ -327,10 +331,14 @@ private void processInputReport(Memory readBuffer, int numBytesRead) {
 			int len = numBytesRead - 1;
 			byte[] inputReport = new byte[len];
 			readBuffer.read(1, inputReport, 0, len);
+			//Log("HB: " + inputReport[1]);
 			if (m_InputReportListener != null) {
 				m_InputReportListener.onInputReport(this, reportID, inputReport, len);
 			}
 		}
+		else {
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/HidDevice.java]: Remove empty block?

**Review by mitza-oci — CHANGES_REQUESTED** [review]

File: src/purejavahidapi/windows/HidDevice.java (hunk 1 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/HidDevice.java:191]: I am trying to play it safe, as I don't have a way to retest. For now I wanted to check it in as it was successfully tested.

File: src/purejavahidapi/windows/HidDevice.java (hunk 2 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/HidDevice.java]: Done

**Review by mitza-oci — APPROVED** [review]

**Pull request merged**